opencti-platform/opencti-front/src/private/Root.tsx

```diff
@@ -141,6 +141,7 @@ const meUserFragment = graphql`
       id
       name
     }
+    is_sensitive_changes_allow
```

Comment:
Need to add feature flag

Reply:
Suggestion of naming : 'can_manage_sensitive_config' ?